tests/e2e/test_package_management.py

```diff
@@ -73,3 +73,25 @@ def test_removing_package(install_package, protostar, libs_path):
 
     assert "Removing starknet_py" in result
     assert "starknet_py" not in listdir(libs_path)
+
+
+@pytest.mark.parametrize("libs_path", ["lib", "deps"])
+@pytest.mark.parametrize("project_new_location", [None, Path("./subproject")])
+@pytest.mark.usefixtures("project_relocator")
+def test_install_remove_install(install_package, protostar, libs_path):
+    install_package()
+    assert "starknet_py" in listdir(libs_path)
+
+    protostar(["--no-color", "remove", "starknet_py"])
+    assert "starknet_py" not in listdir(libs_path)
+
+    install_package()
+    assert "starknet_py" in listdir(libs_path)
+
+
+@pytest.mark.parametrize("libs_path", ["lib", "deps"])
+@pytest.mark.parametrize("project_new_location", [None, Path("./subproject")])
+@pytest.mark.usefixtures("project_relocator")
+def test_install_specified_tag(protostar, libs_path):
+    protostar(["--no-color", "install", "software-mansion/starknet.py@0.6.2-alpha"])
+    assert "starknet_py" in listdir(libs_path)
```

Comment:
Consider checking if the installed package version is actually equal to `0.6.2-alpha`.